src/Microsoft.AspNetCore.Identity/IdentityOptions.cs

```diff
@@ -11,6 +13,18 @@ namespace Microsoft.AspNetCore.Builder
     /// </summary>
     public class IdentityOptions
     {
+        public IdentityOptions(IEnumerable<IConfigureOptions<IdentityOptions>> configureOptions = null)
```

Comment:
I hope so, because we had added `Options.Create` to avoid having to implement `IOptions<>` to return `this` (which was ugly), and now we have this much more horrible pattern... :smile: